background/storage.js

```diff
@@ -350,6 +360,8 @@ function saveStyle(style) {
   let existed;
   let codeIsUpdated;
 
+  style = fixUsoMd5Issue(style);
```

Comment:
We are manipulating an object so there is no need to reassign. Simply use `fixUsoMd5Issue(style)` instead.